src/lib/pages/backoffice/Document/DocumentForms/DocumentEditor/DocumentForm/DocumentForm.js

```diff
@@ -149,7 +149,11 @@ export class DocumentForm extends Component {
         <UrlsField />
         <Subjects />
         <InternalNotes />
-        <Identifiers />
+        <Identifiers
+          vocabularies={{
+            scheme: invenioConfig.vocabularies.document.identifier.scheme,
```

Comment:
Thanks! FIY, it has been also fixed by @topless [here](https://github.com/inveniosoftware/react-invenio-app-ils/pull/58/files#diff-371c74c6ff553ae6117cd64b67c529f3R156) in his PR.